bloom/exceptions.py

```diff
@@ -0,0 +1,32 @@
+#-----------------------------------------------------------------------------#
+#   exceptions.py                                                             #
+#                                                                             #
+#   Copyright (c) 2017, Rajiv Bakulesh Shah, original author.                 #
+#   All rights reserved.                                                      #
+#-----------------------------------------------------------------------------#
+
+
+
+class BloomFilterException(Exception):
+    def __init__(self, memcache=None, key=None, retriable=False):
+        super(BloomFilterException, self).__init__()
+        self.memcache = memcache
+        self.key = key
+        self.retriable = retriable
+
+    def __repr__(self):     # pragma: no cover
+        return '<{} key={} retriable={}>'.format(
+            self.__class__.__name__,
+            self.key,
+            self.retriable,
+        )
+
```

Comment:
Should be two blank lines here!

Reply:
:P

Reply:
[Good call.](https://www.python.org/dev/peps/pep-0008/#blank-lines)  Addressed.